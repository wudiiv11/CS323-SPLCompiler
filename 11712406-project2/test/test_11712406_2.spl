struct Student {
  int name;
};

int main(int c) {
  struct Student tom;
  int i = 0;
  while (i < 10) {
    people[i].name = 3.4;
    tom.friends[i] = i;
    i = i + 1;
  }
  return friend;
}