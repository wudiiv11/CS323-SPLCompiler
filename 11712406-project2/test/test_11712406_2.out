Error type 1 at Line 9: undefined varaible: people
Error type 14 at Line 9: accessing with non-struct variable
Error type 5 at Line 9: unmatching type on both sides of assignment
Error type 14 at Line 10: no such member: friends
Error type 5 at Line 10: unmatching type on both sides of assignment
Error type 1 at Line 13: undefined varaible: friend